projects/baby_driver/scripts/test_can_util.py

```diff
@@ -0,0 +1,53 @@
+""" This Module Tests methods in can_util.c """
+import unittest
+
+from can_util import can_pack
+
+class TestCanPack(unittest.TestCase):
+    """ Test Can Util data_pack function """
+
+    def test_low_bound(self):
+        """ Tests Maximum values """
```

Comment:
You mean tests *minimum* values? Also the standard is to not use spaces around the `"""`, like `"""Tests minimum values"""`